src/Uno/Prism.Uno/PrismApplicationBase.cs

```diff
@@ -0,0 +1,203 @@
+
+        /// <summary>
+        /// Configures the <see cref="IRegionBehaviorFactory"/>. 
+        /// This will be the list of default behaviors that will be added to a region. 
+        /// </summary>
+        protected virtual void ConfigureDefaultRegionBehaviors(IRegionBehaviorFactory regionBehaviors)
+        {
+            regionBehaviors?.RegisterDefaultRegionBehaviors();
+        }
+
+        /// <summary>
+        /// Configures the default region adapter mappings to use in the application, in order
+        /// to adapt UI controls defined in XAML to use a region and register it automatically.
+        /// May be overwritten in a derived class to add specific mappings required by the application.
+        /// </summary>
+        /// <returns>The <see cref="RegionAdapterMappings"/> instance containing all the mappings.</returns>
+        protected virtual void ConfigureRegionAdapterMappings(RegionAdapterMappings regionAdapterMappings)
+        {
+            if (regionAdapterMappings != null)
+            {
+                regionAdapterMappings.RegisterMapping(typeof(Selector), _containerExtension.Resolve<SelectorRegionAdapter>());
+                regionAdapterMappings.RegisterMapping(typeof(ItemsControl), _containerExtension.Resolve<ItemsControlRegionAdapter>());
+                regionAdapterMappings.RegisterMapping(typeof(ContentControl), _containerExtension.Resolve<ContentControlRegionAdapter>());
+            }
+        }
+
+        /// <summary>
+        /// Registers the <see cref="Type"/>s of the Exceptions that are not considered 
+        /// root exceptions by the <see cref="ExceptionExtensions"/>.
+        /// </summary>
+        protected virtual void RegisterFrameworkExceptionTypes()
+        {
+        }
+
+        /// <summary>
+        /// Creates the shell or main window of the application.
+        /// </summary>
+        /// <returns>The shell of the application.</returns>
+        protected abstract UIElement CreateShell();
+
+        /// <summary>
+        /// Initializes the shell.
+        /// </summary>
+        protected virtual void InitializeShell(UIElement shell)
+        {
+            Windows.UI.Xaml.Window.Current.Content = shell;
+        }
+
+        /// <summary>
+        /// Contains actions that should occur last.
+        /// </summary>
+        protected virtual void OnInitialized()
+        {
+            Windows.UI.Xaml.Window.Current.Activate();
+        }
+
+        /// <summary>
+        /// Configures the <see cref="IModuleCatalog"/> used by Prism.
+        /// </summary>
+        protected virtual void ConfigureModuleCatalog(IModuleCatalog moduleCatalog) { }
+
+        /// <summary>
+        /// Initializes the modules.
+        /// </summary>
+        protected virtual void InitializeModules()
+        {
+            IModuleManager manager = _containerExtension.Resolve<IModuleManager>();
+            manager.Run();
+        }
```

Comment:
Please update to use the new extension `PrismInitializationExtensions.RunModuleManager(Container);`